justfile

```diff
@@ -141,6 +141,14 @@ docker-debug:
     # podman builds do not work without --platform set, even though it claims to default to host arch
     docker buildx build . --platform linux/{{ _docker_arch }} --file docker/debug.Dockerfile --tag={{ docker_image }} --progress='{{ DOCKER_PROGRESS }}' --build-arg RESTATE_FEATURES={{ features }} --load
 
+docker-local-fedora:
```

Comment:
Add a comment that this command only works when being executed on a Fedora machine that is the same version as the base image used by `local-fedora.Dockerfile`.